src/beautifiers/php-cs-fixer.coffee

```diff
@@ -24,6 +24,7 @@ module.exports = class PHPCSFixer extends Beautifier
       "fix"
       "--rules=#{options.rules}" if options.rules
       "--config=#{configFile}" if configFile
+      "--allow-risky=#{options.allow_risky}" if options.allow_risky
```

Comment:
Sounds good! I think this is good as is.

> As long as it is not in the config file it will not be included (I am not sure about this)

The default value should be present, so `no`. So as long as `php-cs-fixer --allow-risky=no` is valid, then we are all good.